src/app/components/Footer.tsx

```diff
@@ -127,7 +127,7 @@ export const Footer = () => {
       }
     } else {
       payload = {
-        backgroundColor: appState?.appState?.editorColor,
+        backgroundColor: appState?.appState.editorColor || '#ffffff',
```

Comment:
same logic is used in multiple places, is it possible to save the value in a variable and use that instead? @aatbip

Reply:
Updates in the commit below @saugatmaharjan